README.md

```diff
@@ -34,6 +34,7 @@ To use the action, add a step to your workflow that uses the following syntax.
       ENV_VAR_NAME, secretId2
     name-transformation: (Optional) uppercase|lowercase|none
     parse-json-secrets: (Optional) true|false
+    auto-select-family-attempt-timeout: (Optional) positive integer
```

Comment:
The integration test that covers this is [here](https://github.com/aws-actions/aws-secretsmanager-get-secrets/blob/main/.github/workflows/tests.yml#L136-L151). The secrets are in south Africa and the github runner is in the US. Could you add this `auto-select-family-attempt-timeout` option to the integ test and specify something like 2s so we actually know it passes end to end?